src/features/import/utils/problems/predictProblems.ts

```diff
@@ -133,11 +122,13 @@ export function predictProblems(
             }
           }
           hadImpact = true;
+        } else {
+          isMissingValue = true;
```

Comment:
This made me a bit confused - this will be set to true if any "cell" in the sheet is empty, no matter if it is name/id or not.